containers/nginx-storage/install-dependencies.sh.j2

```diff
@@ -21,11 +21,22 @@ ln -s /usr/lib/libcrypto.so.41 /usr/lib/libcrypto.so
 git clone                                            \
     https://github.com/pgaertig/nginx-big-upload.git \
     /opt/nginx-big-upload                            \
-    --branch v1.4.0                                  \
+    --branch {{ nginx_big_upload_version }}          \
     --depth  1
 
 rm -rf /opt/nginx-big-upload/.git
 
+# Github repository with lua-resty-http module
+git clone                                            \
+    https://github.com/pintsized/lua-resty-http.git  \
+    /opt/lua-resty-http                              \
```

Comment:
Adding a new dependency should be always done in a separate commit.

Reply:
Done.